client/my-sites/activity/activity-log-v2/index.tsx

```diff
@@ -28,10 +36,46 @@ import './style.scss';
 
 const ActivityLogV2: FunctionComponent = () => {
 	const translate = useTranslate();
+	const dispatch = useDispatch();
 
 	const siteId = useSelector( getSelectedSiteId );
 	const filter = useSelector( ( state ) => getActivityLogFilter( state, siteId ) );
 	const logs = useSelector( () => getHttpData( getRequestActivityLogsId( siteId, filter ) ).data );
+	const selectedSiteSlug = useSelector( getSelectedSiteSlug );
+	const siteIsOnFreePlan = useSelector(
+		( state ) =>
+			isFreePlan( get( getCurrentPlan( state, siteId ), 'productSlug' ) ) &&
+			! isVipSite( state, siteId )
+	);
+	const siteHasBackupPurchase = useSelector( ( state ) =>
+		siteHasBackupProductPurchase( state, siteId )
+	);
+
+	const showUpgrade = siteIsOnFreePlan && ! siteHasBackupPurchase;
+	const showFilter = ! showUpgrade;
+
+	const jetpackCloudHeader = showUpgrade ? (
+		<Upsell
+			headerText={ translate( 'Welcome to your site’s activity' ) }
+			bodyText={ translate(
+				'With your free plan, you can monitor the 20 most recent events. A paid plan unlocks more powerful features. You can access all site activity for the last 30 days and filter events by type and date range to quickly find the information you need.'
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **19** times:
`translate( 'With your free plan, you can monitor the 20 most recent events. A paid plan unlocks more powerful features. You can access all site activity for the last 30 days and filter events by type and date range to quickly find the information you need. ' )` *ES Score: 9*